lib/muffin_man/reports/v20210630.rb

```diff
@@ -71,7 +71,14 @@ def get_report_document(report_document_id)
         report_document_id = sandbox ? SANDBOX_REPORT_DOCUMENT_ID : report_document_id
         @local_var_path = "/reports/2021-06-30/documents/#{report_document_id}"
         @request_type = "GET"
-        call_api
+        response = call_api
+        parsed_response=JSON.parse(response.body)
+        report=Net::HTTP.get(URI.parse(parsed_response['url']))
+        unless (parsed_response['compressionAlgorithm']).nil?
+          input = StringIO.new(report)
+          report = Zlib::GzipReader.new(input).read
+        end
+        report
```

Comment:
Sorry, can we actually make this a separate method? It could be called something like `get_parsed_report_document` or `get_report_document_body` or something along those lines, and it would call 
```
response = get_report_document(report_document_id)
parsed_response = JSON.parse(response.body) 
...
```
That way, if someone needs to just get the report document, it's still available via this method.

Reply:
Sounds good.